### PR TITLE
feat(ui): Add StatusChip Component and fix Listing Component column failing with Hooks

### PR DESCRIPTION
**Pull request by bdauria**

### CODE REVIEW

File: src/Snackbar/Error/index.jsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ const ErrorSnackbar = ({ message, open, onClose }) => {
   return (
     <Snackbar
       anchorOrigin={{
-        vertical: 'top',
+        vertical: 'bottom',
```
Comment (kduret) [inline src/Snackbar/Error/index.jsx]: is it the behaviour we want ? (eg: BA form)

File: src/StatusChip/index.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      clickable={clickable}
+      label={label?.toUpperCase()}
+      className={classes.chip}
+      {...rest}
+    />
+  );
+};
+
+export default StatusChip;
+export { StatusCode };
```
Comment (kduret) [inline src/StatusChip/index.tsx]: do we have a rule to order exports ? default one before named exports ?

File: src/Snackbar/Error/index.jsx (hunk 1 above)
Comment (bdauria) [reply src/Snackbar/Error/index.jsx]: I think we should have the same behaviors across all pages for this kind of feature

File: src/StatusChip/index.tsx (hunk 2 above)
Comment (bdauria) [reply src/StatusChip/index.tsx]: We don't have any rule for that yet. What's your opinion, default should be last?

Comment (kduret) [reply src/StatusChip/index.tsx]: yes, IMHO, I think default should be the last one

File: src/Snackbar/Error/index.jsx (hunk 1 above)
Comment (kduret) [reply src/Snackbar/Error/index.jsx]: Yes indeed, but my question was more, "do we want it at the bottom" ?
But I guess yes :)

Comment (bdauria) [reply src/Snackbar/Error/index.jsx]: Yes, the standard position is bottom (that's also what the UV wireframes show)

File: src/StatusChip/index.tsx (hunk 2 above)
Comment (bdauria) [reply src/StatusChip/index.tsx]: Do you mind updating the wiki?

File: src/Listing/index.tsx [hunk 3]
```diff
@@ -429,7 +421,10 @@ const Listing = ({
               })}
               {tableData.length < 1 && (
                 <ListingRow tabIndex={-1}>
-                  <BodyTableCell colSpan={6} align="center">
+                  <BodyTableCell
+                    colSpan={columnConfiguration.length + 1}
```
Comment (Thebarda) [inline src/Listing/index.tsx]: Maybe you should test if checkboxes are displayed through `checkable` before adding 1

Comment (bdauria) [reply src/Listing/index.tsx]: Having an extra colSpan doesn't differ visually, so I don't think a check is necessary.

**Pull request merged**